### PR TITLE
 lab 4 comments

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -51,23 +53,22 @@ ul {
     min-width: 0;
     max-width: 1024px;
     margin: auto;
-    padding: 2em 0em 2em 0em;
+    padding: 0 2em 0 2em;
```
Comment (rmartin-sc) [inline styles/main.css:56]: Incorrect order.  It starts with padding-top and works around clockwise, so the 2nd and 4th entries are left & right padding, respectively

File: styles/main.css [hunk 2]
```diff
@@ -77,7 +78,7 @@ h1, h2, h3, h4 h5, h6 {
     color: transparent;
 }
 footer hr{
-    width: 50%;
+    width: 50vw;
```
Comment (rmartin-sc) [inline styles/main.css:81]: Incorrect unit

File: styles/main.css [hunk 3]
```diff
@@ -51,23 +53,22 @@ ul {
...
-    padding: 2em 0em 2em 0em;
+    padding: 0 2em 0 2em;
     margin-bottom:3em;
-    border-bottom: 1px;
-    border-right: 1px;
-    border-left: 1px;
-    border-style: solid;
+    border-bottom: 1px solid;
+    border-right: 1px solid;
+    border-left: 1px solid;
```
Comment (rmartin-sc) [inline styles/main.css:60]: Missing border-style

File: styles/main.css [hunk 4]
```diff
@@ -20,6 +20,8 @@ a:visited {
 }
 nav ul a:link {
     color: #D49C6A;
+}
+nav ul {
```
Comment (rmartin-sc) [inline styles/main.css:24]: You need to apply the margin and padding to the `ul` directly to affect the list's margin and padding

**Review by RB16014664 — APPROVED** [review]

**Pull request merged**